packages/components/addon/components/hds/tag/index.hbs

```diff
@@ -0,0 +1,26 @@
+<span class={{this.classNames}} ...attributes>
+  {{#if this.onDismiss}}
+    <button class="hds-tag__dismiss" type="button" aria-label="Dismiss {{this.text}}" {{on "click" this.onDismiss}}>
+      <FlightIcon class="hds-tag__dismiss-icon" @name="x" @size="16" @isInlineBlock={{false}} />
```

Comment:
```suggestion
      <FlightIcon class="hds-tag__dismiss-icon" @name="x" @size="16" @isInlineBlock={{false}} @color="var(--token-color-foreground-primary)" />
```

Reply:
Thanks! Moved the declaration to css as we have a `hds-tag__dismiss-icon` that we can leverage.